exporter/exporterhelper/queue_sender_test.go

```diff
@@ -328,6 +328,7 @@ func TestQueuedRetryPersistentEnabled_NoDataLossOnShutdown(t *testing.T) {
 
 	// wait for the item to be consumed from the queue
 	replacedReq.checkNumRequests(t, 1)
+	require.NoError(t, be.Shutdown(context.Background()))
```

Comment:
Use defer, or even better t.Cleanup() to make sure always is executed, immediately after start.

Reply:
Done, used `t.Cleanup()`. 

I found a lot of other `Shutdown` calls in this file identical to what I originally had, but I've left them unchanged for now. I can update them as well if necessary, but I thought it was out of scope for this change.